templates/travel/partials/icon/icon.snip.html

```diff
@@ -0,0 +1,74 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#icon}}
```

Comment:
This does not seem like reusable code to me, partials should only contain reusable stuff (within your templates)

Reply:
We also use it here: https://github.com/ampproject/ampstart/pull/696/files#diff-96f1d334ffc139c20641005d15fb26a3R24